examples/machine-set.yaml

```diff
@@ -19,9 +17,10 @@ spec:
       labels:
         machine.openshift.io/cluster-api-machineset: test-master
         machine.openshift.io/cluster-api-cluster: tb-asg-35
-        machine.openshift.io/cluster-api-machine-role: infra
-        machine.openshift.io/cluster-api-machine-type: master
     spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/master: "infra"
```

Comment:
Is `node-role.kubernetes.io/master: "infra"` a valid role label? I am familiar only with `node-role.kubernetes.io/master: ""` or `node-role.kubernetes.io/worker: ""` forms.

Reply:
Since value is string type and i have seen forms like `node-role.kubernetes.io/master: "true"` as well, I think it depends on implementation that what is treated as valid. 

Alternative could be to use two labels: `node-role.kubernetes.io/master: ""` and `node-role.kubernetes.io/infra: ""` . But since data type allows arbitrary strings, thought of using it instead of increasing label count.

Reply:
it should be `node-role.kubernetes.io/master` and `node-role.kubernetes.io/infra`.